README.rst

```diff
@@ -116,8 +116,7 @@ set up separately):
   * Publish the release on the chosen hosting (GitLab or GitHub, cloud or self-hosted).
   * Publish the released package to the chosen Python package repository.
 
-* Command line interface (optional), with testing, using Cleo_ (default), Click_ or
-  Argparse_.
+* Command line interface (optional), with testing, using Cleo_ (default), Click_.
```

Comment:
Replace the last comma with `or` ?

Reply:
@esciara There was only this very minor comment before being able to close this PR.
Could you check on this ?

Reply:
done